index.coffee

```diff
@@ -16,10 +18,18 @@ exports = module.exports = (opts, cb) ->
 	method = (opts.method ? 'get').toLowerCase()
 	if logEnabled
 		console.log('requestmock:', url, method, opts)
-	if handlers[url]?[method]?
-		return handlers[url][method](opts, cb)
+
+	# First look for an exact match, then scan for a matching regex handler.
+	# If no matches are found, call the default handler
+	if fixed_url_handlers[url]?[method]?
```

Comment:
Why underscore_case ?

Reply:
No reason at all, thanks for spotting - fixed.